src/main/java/org/openstreetmap/atlas/checks/maproulette/MapRouletteUploadCommand.java

```diff
@@ -120,13 +120,13 @@ protected void execute(final CommandMap commandMap,
                         {
                             try
                             {
-                                final Challenge challenge = this
-                                        .getChallenge(task.getChallengeName(), instructions);
+                                final Challenge challenge = this.getChallenge(check, instructions);
+                                final Challenge mockChallenge = new Challenge(challenge);
```

Comment:
Could this be combined with the line above, or would that break where you call challenge.getName() below?

Reply:
Yeah that could be done, but multiple variable initialization actually throws Sonar error S1659 (declare variables on separate lines)